src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -69,9 +69,9 @@ public async Task RunApplicationAsync(CancellationToken cancellationToken = defa
             PrepareServices();
             PrepareContainers();
             PrepareExecutables();
-            PrepareProxylessServices();
+            // PrepareProxylessServices();
 
-            await CreateContainerSingletonsAsync(cancellationToken).ConfigureAwait(false);
+            // await CreateContainerSingletonsAsync(cancellationToken).ConfigureAwait(false);
 
```

Comment:
@bwateratmsft @karolz-ms I ended up disabling this to make the PR work. Any guidance here on how this is supposed to work would be appreciated.

Reply:
We had a conversation about this in https://github.com/dotnet/aspire/pull/886. Long story short, we need more time and design to enable this.

Reply:
FYI unused code was causing code analysis warnings. I've removed it here - https://github.com/dotnet/aspire/pull/1286. Source is still available in the file history 😄

Reply:
Thank you!